src/test/java/org/apache/commons/lang3/ClassUtilsTest.java

```diff
@@ -1202,6 +1206,56 @@ public void testGetInnerClass() throws ClassNotFoundException {
         assertEquals( Inner.DeeplyNested.class, ClassUtils.getClass( "org.apache.commons.lang3.ClassUtilsTest$Inner.DeeplyNested" ) );
     }
 
+    @Test
+    public void testGetBaseClassesExtends() throws Exception {
+        List<AnotherParent> classes = ClassUtils.getBaseClasses(AnotherParent.class, "org.apache.commons.lang3.reflect.testbed");
+        assertTrue(classes.size() > 0);
+    }
+
+    @Test
+    public void testGetBaseClassesAbstract() throws Exception {
+        List<AbstractConcurrentInitializerTest> classes = ClassUtils.getBaseClasses(AbstractConcurrentInitializerTest.class, "org.apache.commons.lang3.concurrent");
+        assertTrue(classes.size() > 0);
```

Comment:
should validate each element of the container for better accuracy.

Reply:
This is a good idea